scripts/pre-push.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+GRAY='\033[30m'  
+RED='\033[31m' 
+YELLOW='\033[33m'
+NC='\033[0m' 
+
+FILES=$(git diff origin/master...HEAD --name-only --diff-filter=d | grep \\.ts)
```

Comment:
we should for sure scan also `tsx` files. Probably also `scss`.

Reply:
yep

Reply:
added tsx + scss 👍